tox.ini

```diff
@@ -4,6 +4,7 @@ envlist = py{37,38,39,310}-{pyqt5,pyside2,pyside6,pyqt6}, linting
 [testenv]
 deps=
     pytest
+    qtpy
```

Comment:
This should not be needed because `qtpy` is already a dependency of `pytest-qt` itself (now that I noticed it, `pytest` above is not needed either).

Reply:
IIRC tests were failing without it

Reply:
Weird, can you double check? `qtpy` is already a dependency for the package, we should not have to duplicate it here... :thinking:

Reply:
Ahh in case you were already running `tox`, try passing `-r` for it to create a fresh environment.